.devcontainer/devcontainer.json

```diff
@@ -34,6 +22,7 @@
     "eamodio.gitlens",
     "mhutchie.git-graph",
     "ms-python.python",
+    "ms-python.black-formatter",
```

Comment:
What is the difference between installing the extension and [installing `black` directly](https://github.com/cal-itp/benefits/blob/dev/.devcontainer/Dockerfile#L16) as part of the devcontainer build? My understanding is the `black` formatting has already been working for a while now...

Reply:
`black` is installed as a package, but wasn't actually being run from the editor. The extension does the latter. Steps to reproduce, without this pull request:

1. Modify a Python file in a way that should be undone with formatting (adding whitespace or whatever).
1. Save — the file doesn't change.
1. From the command palette, run `Format Document`.

   <img width="744" alt="Screen Shot 2022-06-13 at 1 18 18 PM" src="https://user-images.githubusercontent.com/86842/173409574-cca43b45-80e0-4389-ac63-b4f77e6b2e25.png">

1. This dialog appears:

   <img width="269" alt="Screen Shot 2022-06-13 at 1 18 26 PM" src="https://user-images.githubusercontent.com/86842/173409748-0440f606-9d33-4b1e-b8fc-d9ea275aa0aa.png">

Reply:
I don't see this behavior on the existing `dev`

1. Modify a Python file
2. Save - the file is formatted with `black`

![save-black](https://user-images.githubusercontent.com/1783439/173413614-dfd0050a-d6a8-4818-8f1a-52c1610f4ecc.gif)

Reply:
Weird. Can you confirm you don't have the `black` extension installed? @machikoyasuda @angela-tran Do you get formatting on save, or no?

Reply:
Confirmed

![image](https://user-images.githubusercontent.com/1783439/173414196-a5574ba5-bcc0-48c1-b4ac-c01bd8e30948.png)

Reply:
@afeld Formatting on save has been working for me as well. I wonder if there might be some difference with Macs?

Reply:
Mind trying rebuilding on this branch? If that works for you all, mind if we add the extension, even if it only makes a difference for me (for some reason)?

Reply:
There are some troubleshooting tips here: https://code.visualstudio.com/docs/python/editing#_troubleshooting-formatting

Reply:
I can also rebuild on your branch and make sure that having the extension would be ok

Reply:
I followed these steps:


1. Modify a Python file in a way that should be undone with formatting (adding whitespace or whatever).
1. Save — the file doesn't change.
1. From the command palette, run `Format Document`.
1. I get a different dialog
![image](https://user-images.githubusercontent.com/25497886/173452575-1cde0178-5f67-489d-8d90-ab8fa7bf63bc.png)

When I click "Configure" and choose "Black Formatter", this is added to `.vscode/settings.json`, and code is formatted on save:
```
  "[python]": {
    "editor.defaultFormatter": "ms-python.black-formatter"
  }
```

Reply:
Asked for clarification: https://github.com/microsoft/vscode-black-formatter/issues/82